examples/next/proxied-sitemap-apollo/README.md

```diff
@@ -0,0 +1,116 @@
+
+## Setup
+
+### Environment Variables
+
+Create or update your .env.local file with:
+```
+NEXT_PUBLIC_WORDPRESS_URL=http://your-wordpress-site.com
+NEXT_PUBLIC_FRONTEND_URL=http://your-nextjs-site.com
+```
+
+## How It Works
+
+Proxying Sitemap Requests:
+
+1. When a request for a sitemap (e.g., `/wp-sitemap-posts-post-1.xml`) is made on the Next.js frontend, the Next.js server fetches the corresponding sitemap from WordPress.
+
+2. The bundled plugin automatically replaces any URLs pointing to the WordPress backend domain (e.g., http://your-wordpress-site.com) with URLs pointing to the frontend domain (e.g., http://your-nextjs-site.com).
+
+3. The transformed sitemap is returned to the client with the correct URLs pointing to the frontend domain and the appropriate Content-Type header (application/xml).
+
+# Running the example with wp-env
+
+## Prerequisites
+
+**Note** Please make sure you have all prerequisites installed as mentioned above and Docker running (`docker ps`)
+
+## Setup Repository and Packages
+
+- Clone the repo `git clone https://github.com/wpengine/hwptoolkit.git`
+- Install packages `cd hwptoolkit && pnpm install`
+- Setup a .env file under `examples/next/proxied-sitemap-apollo/example-app` with `NEXT_PUBLIC_WORDPRESS_URL=http://localhost:8888`
+e.g.
+
+```bash
+echo "NEXT_PUBLIC_WORDPRESS_URL=http://localhost:8888\\nNEXT_PUBLIC_FRONTEND_URL=http://localhost:3000" > examples/next/proxied-sitemap-apollo/example-app/.env
+```
+
+## FRONTEND_URL (Optional Customization)
+The FRONTEND_URL is automatically set by default to `http://localhost:3000`. This is the URL to which your sitemaps will point. If you want to change it to a different frontend domain (for example, if your Next.js site is hosted elsewhere), you can define the FRONTEND_URL constant in your wp-config.php file.
+
+To define it manually, add the following line to your wp-config.php:
+
+```php
+define( 'FRONTEND_URL', 'http://your-nextjs-site.com' );
+```
+If you don't define it explicitly, the default value of http://localhost:3000 will be used.
+
+## Build and start the application
+
+- `cd examples/next/proxied-sitemap-apollo`
+- Then run `pnpm example:build` will build and start your application. 
+- This does the following:
+    - Unzips `wp-env/uploads.zip` to `wp-env/uploads` which is mapped to the wp-content/uploads directory for the Docker container.
+    - Starts up [wp-env](https://developer.wordpress.org/block-editor/getting-started/devenv/get-started-with-wp-env/)
+    - Imports the database from [wp-env/db/database.sql](wp-env/db/database.sql)
+    - Install Next.js dependencies for `example-app`
+    - Runs the Next.js dev script
+
+Congratulations, WordPress should now be fully set up.
+
+| Frontend | Admin                        |
+|----------|------------------------------|
+| [http://localhost:3000/](http://localhost:3000/) | [http://localhost:8888/wp-admin/](http://localhost:8888/wp-admin/) |
```

Comment:
@theodesp 

One minor suggestion is to add some links for the sitemap here too e.g. http://localhost:3000/sitemap.xml and http://localhost:8888/wp-sitemap-posts-post-1.xml